fast64_internal/sm64/sm64_constants.py

```diff
@@ -1802,213 +1802,213 @@ def __init__(self, geoAddr, level, switchDict):
 
 marioAnimations = [
 #   ( Adress, "Animation name" ),
-    ( 5162640, "0 - Climb up ledge (right leg going up)"),
-    ( 5165520, "1 - Thrown from painting"),
-    ( 5165544, "2 - Blown by wind"),
-    ( 5172396, "3 - Dying on ground"),
+    ( 5162640, "0 - Slow ledge grab"),
```

Comment:
```suggestion
    ( 5162640, "0 - Slow ledge climb up"),
```
something like this? or is this a grab when walking off a ledge